Documentation/design/resource_dep.dot

```diff
@@ -79,11 +79,16 @@ strict digraph resource {
     kubeconfig_kubelet [label="kubeconfig-kubelet"];
   }
 
-
-  // Output after cluster launch
+  // Terraform tfvar file.
   {
     node [style=filled,color=gold];
-    terraform_state [label="*.tfstate"];
+    terraform_tfvars [label="terraform.tfvars"];
+  }
+
+  // OpenShift cluster.
+  {
+    node [style=filled, color=lemonchiffon];
+    openshift_cluster [label="cluster"];
```

Comment:
The distinction between `cluster` and `openshift_cluster` is not clear to me, especially with `openshift_cluster` -> cluster`.  Can we get more descriptive names for these?  Or maybe drop `cluster` and rename `openshift_cluster` -> `cluster`?